aries-site/src/examples/foundation/accessibility/SectionCards.js

```diff
@@ -0,0 +1,88 @@
+import React, { useContext } from 'react';
+import { Box, Button, Grid, ResponsiveContext } from 'grommet';
+import { LinkCard } from '../../../components';
+
+const guides = [
+  {
+    title: 'Complex Images in Software',
+    link:
+      'https://hpe.sharepoint.com/:b:/s/F5/CTO/Office/Accessibility/EYWU1LUDz8xLhVu-omWEYpwB9ZcpjszrA0hMC7vjSx5jiA?e=EctwhC',
+  },
+  {
+    title: 'Links and HyperLinks',
+    link:
+      'https://hpe.sharepoint.com/:b:/s/F5/CTO/Office/Accessibility/EXN7DvqsNvBEmfKD1ScESIMBBqFCSjtygi79PSQwJ_jWFQ?e=YnAjhe',
+  },
+  {
+    title: 'Evaluation Tools and Toolbars',
+    link:
+      'https://hpe.sharepoint.com/:b:/s/F5/CTO/Office/Accessibility/EbNhmRbHK6BCsiyLbRRNaE0BQZQlM3D2lVM60cL3nPR67g?e=dgfrFs',
+  },
+  {
+    title: 'Documentation to Inform and Enable',
+    link:
+      'https://hpe.sharepoint.com/:b:/s/F5/CTO/Office/Accessibility/EbKbkBd8q2NLkmc2KKz5j_4BnuVCs6voFzhH2KwKuqFFIQ?e=ZcMBfn',
+  },
+];
+
+const videos = [
+  {
+    title: 'Designing Accessible Software',
+    link:
+      'https://hpe.sharepoint.com/portals/hub/_layouts/15/PointPublishing.aspx?app=video&p=p&chid=33203c05%2D71a0%2D49cf%2D8daf%2De7bc7c0c920d&vid=7f47a741%2D9bfe%2D40d0%2Db084%2D01f41fe12cc0&from=1',
+  },
+  {
+    title: 'Layout and Page Structure',
+    link:
+      'https://hpe.sharepoint.com/portals/hub/_layouts/15/PointPublishing.aspx?app=video&p=p&chid=33203c05-71a0-49cf-8daf-e7bc7c0c920d&vid=1edcc9f5-9192-49d3-a139-6b695a098695&from=1',
+  },
+  {
+    title: 'Understanding Keyboard Design',
+    link:
+      'https://hpe.sharepoint.com/portals/hub/_layouts/15/PointPublishing.aspx?app=video&p=p&chid=33203c05%2D71a0%2D49cf%2D8daf%2De7bc7c0c920d&vid=5989e0ea%2D716e%2D42f0%2Da4f0%2Dca9944b20c07&from=1',
+  },
+  {
+    title: 'Tab Order and on-Screen Focus',
+    link:
+      'https://hpe.sharepoint.com/:b:/s/F5/CTO/Office/Accessibility/EbKbkBd8q2NLkmc2KKz5j_4BnuVCs6voFzhH2KwKuqFFIQ?e=ZcMBfn',
+  },
+];
+
+const SectionCards = ({ items, type }) => {
+  const size = useContext(ResponsiveContext);
+  const href =
```

Comment:
Similar to my previous comment, how can we make this be more "dumb" and let the caller supply the href? 

What happens when we like this pattern and want to use this section of cards for a bunch of Figma tutorial videos also? We don't want to have to manage the data and mapping logic within SectionCards. Instead, let's make SectionCards be "super accommodating" (liking that phrase better than 'dumb"), so much so as "Just give me: 1) a list of objects -- I don't care what they are or represent -- they just need to have a 'title', 'link/url', [and maybe an 'icon'], and 2) an object for "see all" which has a call-to-action label and an url.

This file then could be moved under 'src/components/cards'... then the data plus the 
```
export const GuideSection = () => <SectionCards type="guides" items={guides} />;
export const VideoSection = () => <SectionCards type="videos" items={videos} />;
```

could be small files living under accessibility, while the rest of the site could consume this pattern as well.